README.md

```diff
@@ -1 +1,35 @@
-# Multi-Compartment-Segmentation
\ No newline at end of file
+# Multi Compartment Segmentation
+
+Multi compartment segmentation is a [detectron2](https://github.com/facebookresearch/detectron2) based panoptic segmentation model that is implemented as a pipeline in [Digital Slide Archive](https://athena.rc.ufl.edu/) (DSA) developed and maintained by [CMI lab](https://cmilab.nephrology.medicine.ufl.edu/) at the University of Florida. This DSA plugin allows users to segment a kidney Whole Slide Image (WSI) into 6 compartments, namely: Interstitium,medulla, glomerulus, sclerotic glomerulus, tubule, and arterial tree.
+
+## Quick Start Guide
+
+This DSA plugin allows users to run image analysis jobs through the HistomicsUI by the following steps:
+
+1. Create an account at [DSA](https://athena.rc.ufl.edu/) and log in with your credentials. Please contact Sayat Mimar at sayat.mimar@ufl.edu for obtaining a DSA account.
+2. Upload a WSI in a folder either unders Collections tab or in your User directory
+3. Open the image in HistomicsUI by clicking the arrow icon:
+
+<img src="figures/fig1.png"  width="60%" height="30%">
+
+4. From **Analyses** tab on the top right in HistomicsUI select: sarderlab/ComPRePS/segmentation/MultiCompartmentSegment
+
+<img src="figures/fig2.png"  width="60%" height="30%">
+
+5. Populate the fields in the tab that appears on the left side of the page: Select the Base Directory as the directory where your WSI is located and select the Model File from the Collections/models/segmentation_models/Multi_compartment_model with an available pre train model and click Sumbit botton.
+
+<img src="figures/fig3.png"  width="60%" height="30%">
+
+6. You can see the logs of the running job and the segmentation progress by clicking the job that you just started from the Jobs Section.
```

Comment:
@sayatmimar : can you see the logs of your running jobs if you are not an admin (like any external users)? I am trying to remember, when I was not admin to Athena I couldn't see the logs of the running jobs. However, it could be the jobs that other people started that I couldn't see. It is worth exploring and pls. verify.

Reply:
@anindya-paul you're right good catch, logs are not visible for non-admin users. I'm removing this part.